### PR TITLE
Sanitize docs for release references

### LINKED ISSUE #816: Remove all references to Radius being in "Private Release" or "Private Preview"
**URL of the docs page**
](https://edge.radapp.dev/community/overview/)

**How is it currently worded?**
There are multiple references to "private release" and "public release: on this page.  We shoudl remove all references to private and public release at this point as that distinction in the docs adds no value.
We need to fix this page and scrub all docs to remove these references now so we are not scrambling to do this before 10/18

Example1 "Welcome to the Radius community. We are currently in a private release phase which is open to Microsoft FTEs and select partners. The team is working on a public preview release which will be announced on this page."

Example2 Every month we host a community meeting to showcase new features, review upcoming milestones, and engage in a Q&A. Anyone from the Radius Private Preview community can participate, present a topic, or host. All are welcome!

**How should it be worded?**
Ex1 should just say "Welcome to the Radius community!" 
Ex2 should just say "Every month we host a community meeting to showcase new features, review upcoming milestones, and engage in a Q&A. Anyone from the Radius community can participate, present a topic, or host. All are welcome!"

**Screenshots**

**Additional context**

[AB#9676](https://dev.azure.com/azure-octo/e61041b4-555f-47ae-95b2-4f8ab480ea57/_workitems/edit/9676)

### PR DESCRIPTION
**Pull request by Reshrahim**
Thank you for helping make the Radius documentation better!

**Please follow this checklist before submitting:**

- [ ] [Read the contribution guide](https://docs.radapp.dev/contributing/docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Sanitize docs for 
- reference to private/public release, preview, production, alpha, beta
- references to old terminologies (link, connectors, quickstarts, reference-apps)

### Summary
📚🆙🚀

This pull request updates the Radius documentation to reflect the latest API version, the new content types, and the versioning policy. It also fixes a misleading button text in the eShop tutorial.

> _`docTypes` expand_
> _Radius API evolves_
> _No more alpha stage_

### Walkthrough
* Update the list of document types for contributing to the Radius documentation ([link](https://github.com/radius-project/docs/pull/837/files?diff=unified&w=0#diff-67bd8967b394fb430efd51387bf2dfae7d9d611d2494ba954c6f30e92d33fe35L207-R210))
* Change the API version to the latest preview version in the Postman guide (`docs/content/guides/operations/control-plane/howto-postman/index.md`, [link](https://github.com/radius-project/docs/pull/837/files?diff=unified&w=0#diff-7615049369926e020ae7e22d2bbb39f8dbd08b67ceebeb045b1891ed4b715778L103-R103))
* Remove the word "alpha" from the versioning policy overview (`docs/content/guides/operations/versioning.md`, [link](https://github.com/radius-project/docs/pull/837/files?diff=unified&w=0#diff-82ad14818de3aa3b8f475583673ba6349c4dc48af4cd8009ef9c245d02342b6bL12-R12))
* Fix the button text for the eShop sample in the tutorial (`docs/content/tutorials/eshop/_index.md`, [link](https://github.com/radius-project/docs/pull/837/files?diff=unified&w=0#diff-83310ec79cae67182b24d39ac67ff2d84d811a55abc2bac26e4637781a8f0830L41-L40))

## Issue reference
Fixes: #816

### CODE REVIEW

**Comment by AaronCrawfis** [comment]
Could you target 0.26 instead of edge please?

File: docs/content/guides/operations/control-plane/howto-postman/index.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ Next, you'll need to get the control plane API endpoint and use Postman to inter
 1. Set the URL to the following, which will [retrieve all your resource groups]({{< ref api-ucp >}}):
     
     ```
-    <your-cluster-control-plane>/apis/api.ucp.dev/v1alpha3/planes/radius/local/resourcegroups?api-version=2022-09-01-privatepreview
+    <your-cluster-control-plane>/apis/api.ucp.dev/v1alpha3/planes/radius/local/resourcegroups?api-version=2023-10-01-preview
```
Comment (AaronCrawfis) [inline docs/content/guides/operations/control-plane/howto-postman/index.md:103]: Good catch, thanks!

File: docs/content/guides/operations/versioning.md [hunk 2]
```diff
@@ -9,6 +9,6 @@ category: "Overview"
 
 ## Overview
 
-Radius is currently in an early alpha preview and does not offer backward compatibility with previous releases. Breaking changes may happen between releases and we recommend doing a fresh installation of the latest version of Radius after every release.
+Radius does not offer backward compatibility with previous releases. Breaking changes may happen between releases and we recommend doing a fresh installation of the latest version of Radius after every release.
```
Comment (AaronCrawfis) [inline docs/content/guides/operations/versioning.md]:
```suggestion
Radius does not currently offer backward compatibility with previous releases. Breaking changes may happen between releases and we recommend doing a fresh installation of the latest version of Radius after every release.
```

**Comment by Reshrahim** [comment]
> Could you target 0.26 instead of edge please?

yes updated.

**Review by AaronCrawfis — APPROVED** [review]

**Pull request merged**